src/toady/command_utils.py

```
@@ -0,0 +1,84 @@
+    This decorator catches ToadyError exceptions and handles them
+    using the unified error handling system.
+
+    Args:
+        func: The command function to wrap.
+
+    Returns:
+        Wrapped function with error handling.
+    """
+
+    @functools.wraps(func)
+    def wrapper(*args: Any, **kwargs: Any) -> Any:
```

Pylint (3.3.7)

[refactor] Either all return statements in a function should return an expression, or none of them should.

(R1710)